print_debug_info.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+echo $?
```

Comment:
FYI:-
In travis each command in script is processed by a special bash function. This function manipulates $? to produce logs suitable for display. Consequently, you should not rely on the value of $? in script section to alter the build behavior.
https://github.com/travis-ci/travis-ci/issues/3771